controllers/statusreport/statusreport_adapter.go

```diff
@@ -247,17 +248,24 @@ func (a *Adapter) EnsureSnapshotFinishedAllTests() (controller.OperationResult,
 // list of integrationTestScenarios.
 func (a *Adapter) determineIfAllIntegrationTestsFinishedAndPassed(integrationTestScenarios *[]v1beta2.IntegrationTestScenario, testStatuses *intgteststat.SnapshotIntegrationTestStatuses) (bool, bool) {
 	allIntegrationTestsFinished, allIntegrationTestsPassed := true, true
+	integrationTestsFinished := len(*integrationTestScenarios)
+	integrationTestsPassed := len(*integrationTestScenarios)
+
 	for _, integrationTestScenario := range *integrationTestScenarios {
 		integrationTestScenario := integrationTestScenario // G601
 		testDetails, ok := testStatuses.GetScenarioStatus(integrationTestScenario.Name)
 		if !ok || !testDetails.Status.IsFinal() {
 			allIntegrationTestsFinished = false
+			integrationTestsFinished--
 		}
 		if ok && testDetails.Status != intgteststat.IntegrationTestStatusTestPassed {
 			allIntegrationTestsPassed = false
+			integrationTestsPassed--
 		}
 
 	}
+	a.logger.Info(fmt.Sprintf("%d out of %d integration tests finished", integrationTestsFinished, len(*integrationTestScenarios)))
+	a.logger.Info(fmt.Sprintf("%d out of %d integration tests passed", integrationTestsPassed, len(*integrationTestScenarios)))
```

Comment:
```suggestion
	a.logger.Info(fmt.Sprintf("%d out of %d required integration tests finished", integrationTestsFinished, len(*integrationTestScenarios)))
	a.logger.Info(fmt.Sprintf("%d out of %d required integration tests passed", integrationTestsPassed, len(*integrationTestScenarios)))
```

Reply:
I deliberately left this out because the function is otherwise agnostic as to whether or not the set of tests it receives are required.  I'm fine with changing the log statement but if so we should also rename the function to `determineIfAllRequiredIntegrationTestsFinishedAndPassed`. Wdyt?

Reply:
so if you want to have function really agnostic, it should return number of finished and number of passed tests and caller with context should log what exactly was finished